component/auth/register.jsx

```diff
@@ -41,7 +39,6 @@ function RegisterPage() {
           Number(err?.response?.data?.inner?.length) > 0
```

Comment:
if and else statement can be replaced by negating other login first and using return statement